components/Frame/index.js

```diff
@@ -117,8 +119,11 @@ const Frame = ({
     })
   }, [hasSecondaryNav])
   return (
-    <div {...(footer || inNativeApp ? styles.bodyGrowerContainer : undefined)}>
-      {/* body growing only needed when rendering a footer */}
+    <ColorContextProvider
```

Comment:
Had some uncertainty when merging this (conflict) as it was recently changed. @tpreusse anything catching your eye?